src/__tests__/sequencerPatch.test.ts

```diff
@@ -0,0 +1,28 @@
+import { expect, test, describe } from "bun:test";
+import { createSequencerPatch } from "../patches/sequencerPatch";
+import { CircuitValidator } from "../validator";
+
+describe("SequencerPatch", () => {
+  test("creates patch with valid circuit keys", () => {
+    const patch = createSequencerPatch();
+    const circuits = (patch as any)._circuits;
```

Comment:
did `patch` has `_circuits`? i think we should impl `getCircuits` method in `Patch` class